server/data-import/get-other-services-csv.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const _ = require('lodash')
+const config = require('config')
+const fs = require('fs')
+const Log = require('log')
+const mongoose = require('mongoose')
+const P = require('bluebird')
+const stringify = require('csv-stringify')
+
+const pregnancyCenterServices = require('../pregnancy-centers/pregnancy-center-services')
+const PregnancyCenterModel = require('../pregnancy-centers/schema/mongoose-schema')
+
+mongoose.Promise = require('bluebird')
+const log = new Log('info')
+
+// TODO: Error handling
+const startDatabase = P.coroutine(function *startDatabase() {
+	yield mongoose.connect(config.mongo.connectionString)
+
+	log.info('Connected to database')
+})
+
+startDatabase()
+
+const mapYesNo = (b) => {
```

Comment:
Would you mind just making this variable name a little more clear? I know that it's just checking the boolean value, but it would be helpful to have a slightly more useful name 😄

Reply:
Yup!